target/start-mailserver.sh

```diff
@@ -49,6 +49,9 @@ DEFAULT_VARS["SPAMASSASSIN_SPAM_TO_INBOX"]="${SPAMASSASSIN_SPAM_TO_INBOX:=0}"
 DEFAULT_VARS["MOVE_SPAM_TO_JUNK"]="${MOVE_SPAM_TO_JUNK:=1}"
 DEFAULT_VARS["VIRUSMAILS_DELETE_DELAY"]="${VIRUSMAILS_DELETE_DELAY:=7}"
 DEFAULT_VARS["NETWORK_INTERFACE"]="${NETWORK_INTERFACE:="eth0"}"
+DEFAULT_VARS["ENABLE_SHARED_INBOX"]="${DOVECOT_ENABLE_INBOX_SHARING:=0}"
+DEFAULT_VARS["NAMESPACE_SEPARATOR_CLAUSE"]="separator = ${DOVECOT_NAMESPACE_SEPARATOR}"
```

Comment:
`DEFAULT_VARS["DOVECOT_NAMESPACE_SEPARATOR_CLAUSE"]="separator = ${DOVECOT_NAMESPACE_SEPARATOR}"`

Reply:
What's up with this one? The namespace separator configuration item may be `separator =` (that's how it is right now), or e.g. `separator = /`.

Reply:
I've just added "Dovecot" to make it more clear/consistent:

`DEFAULT_VARS["NAMESPACE_SEPARATOR_CLAUSE"]` becomes
`DEFAULT_VARS["DOVECOT_NAMESPACE_SEPARATOR_CLAUSE"]`

Reply:
OK that makes sense, sorry, I have overlooked that.

Reply:
Is this resolved?